apps/dashboard/src/components/contract-functions/contract-functions.tsx

```diff
@@ -104,6 +104,7 @@ export const ContractFunctionsOverview: React.FC<ContractFunctionsOverview> = ({
           )}
           {(sources || abi) && (
             <TabPanel>
+              {/* @ts-expect-error This won't cause any error and will auto-resolve itself once we have fully migrated to v5 */}
```

Comment:
@jnsdls I don't like doing this but the ABI thingy is everywhere and we have to remove it bit by bit